scripts/linters/pylint_extensions.py

```diff
@@ -1484,31 +1484,63 @@ class RestrictedImportChecker(checkers.BaseChecker):
             'domain layer and controller layer respectively.'),
     }
 
-    def visit_import(self, node):
-        """Visits every import statement in the file.
+    options = (
+        (
+            'forbidden-imports',
+            {
+                'default': ('core.storage:core.domain',),
+                'type': 'csv',
+                'metavar': '<comma separated list>',
+                'help': 'List of disallowed imports.'
```

Comment:
Add details about the format of the item in the list.

Reply:
Done.